client/src/app/components/providers/WormholeProvider.tsx

```diff
@@ -181,6 +182,16 @@ export default function WormholeProvider(props: Props) {
     );
   }, []);
 
+  const noSleep = useRef(new nosleep());
+  useEffect(() => {
+    // Prevent mobile browsers from sleeping during transfer
+    if (fileMeta && !done) {
```

Comment:
Is this activated only during the transfer going on or also when code is generated. (it should be on both stages be activated)

Reply:
`fileMeta` contains the file information which is non-null after selecting a file, entering the code, and during download. So this is activated at those times.